### PR TITLE
Make FP reg operand notation consistent

### LINKED ISSUE #395: Floating point loads inconsistent mnemonic
Found in `src/insns/load_32bit_fp.adoc`

Should these both be `frd` or `rd` for consistency?

image

### PR DESCRIPTION
**Pull request by andresag01**
Fixes #395 using the same notation from the RISC-V unprivileged ISA spec

### CODE REVIEW

**Comment by tariqkurd-repo** [comment]
you've changed the assembly syntax which does use the f notation
it is a bugbear of mine that risc-v use integer src/dsts for fpu instructions, it makes their spec much harder to use

**Review by tariqkurd-repo — COMMENTED** [review]
can we double check the RISC-V assembly syntax? I'm pretty sure it uses f numbers

File: src/insns/load_16bit_fp_dp.adoc [hunk 1]
```diff
@@ -11,16 +11,16 @@ Synopsis::
 Double precision floating point loads (C.FLD, C.FLDSP), 16-bit encodings
 
 {cheri_cap_mode_name} Mnemonic (RV32)::
-`c.fld frd', offset(cs1'/csp)`
+`c.fld rd', offset(cs1'/csp)`
```
Comment (tariqkurd-repo) [inline src/insns/load_16bit_fp_dp.adoc:14]:
```suggestion
`c.fld frd', offset(cs1'/csp)`
```

File: src/insns/load_16bit_fp_dp.adoc [hunk 2]
```diff
@@ -11,16 +11,16 @@ Synopsis::
...
 
 {cheri_cap_mode_name} Expansion (RV32)::
-`fld frd', offset(csp)`
+`fld rd', offset(csp)`
```
Comment (tariqkurd-repo) [inline src/insns/load_16bit_fp_dp.adoc:17]:
```suggestion
`fld frd', offset(csp)`
```

File: src/insns/load_16bit_fp_dp.adoc [hunk 3]
```diff
@@ -11,16 +11,16 @@ Synopsis::
...
 
 {cheri_int_mode_name} Mnemonic::
-`c.fld fs2, offset(rs1'/sp)`
+`c.fld rs2, offset(rs1'/sp)`
```
Comment (tariqkurd-repo) [inline src/insns/load_16bit_fp_dp.adoc:20]:
```suggestion
`c.fld fs2, offset(rs1'/sp)`
```

File: src/insns/load_16bit_fp_dp.adoc [hunk 4]
```diff
@@ -11,16 +11,16 @@ Synopsis::
...
 
 {cheri_int_mode_name} Expansion::
-`fld fs2, offset(rs1'/sp)`
+`fld rs2, offset(rs1'/sp)`
```
Comment (tariqkurd-repo) [inline src/insns/load_16bit_fp_dp.adoc:23]:
```suggestion
`fld fs2, offset(rs1'/sp)`
```

File: src/insns/load_32bit_fp.adoc [hunk 5]
```diff
@@ -17,9 +17,9 @@ Synopsis::
 Floating point loads (FLD, FLW, FLH), 32-bit encodings
 
 {cheri_cap_mode_name} Mnemonics::
-`fld frd, offset(cs1)` +
-`flw frd, offset(cs1)` +
-`flh frd, offset(cs1)`
+`fld rd, offset(cs1)` +
```
Comment (tariqkurd-repo) [inline src/insns/load_32bit_fp.adoc:20]:
```suggestion
`fld frd, offset(cs1)` +
```

File: src/insns/load_32bit_fp.adoc [hunk 6]
```diff
@@ -17,9 +17,9 @@ Synopsis::
...
+`flw rd, offset(cs1)` +
```
Comment (tariqkurd-repo) [inline src/insns/load_32bit_fp.adoc:21]:
```suggestion
`flw frd, offset(cs1)` +
```

File: src/insns/load_32bit_fp.adoc [hunk 7]
```diff
@@ -17,9 +17,9 @@ Synopsis::
...
+`flh rd, offset(cs1)`
```
Comment (tariqkurd-repo) [inline src/insns/load_32bit_fp.adoc:22]:
```suggestion
`flh frd, offset(cs1)`
```

File: src/insns/store_16bit_fp_dp.adoc [hunk 8]
```diff
@@ -10,20 +10,20 @@ Synopsis::
 Double precision floating point stores (C.FSD, C.FSDSP), 16-bit encodings
 
 {cheri_cap_mode_name} Mnemonics (RV32)::
-`c.fsd fs2, offset(cs1')` +
-`c.fsd fs2, offset(csp)`
+`c.fsd rs2, offset(cs1')` +
```
Comment (tariqkurd-repo) [inline src/insns/store_16bit_fp_dp.adoc:13]:
```suggestion
`c.fsd fs2, offset(cs1')` +
```

File: src/insns/store_16bit_fp_dp.adoc [hunk 9]
```diff
@@ -10,20 +10,20 @@ Synopsis::
...
+`c.fsd rs2, offset(csp)`
```
Comment (tariqkurd-repo) [inline src/insns/store_16bit_fp_dp.adoc:14]:
```suggestion
`c.fsd fs2, offset(csp)`
```

File: src/insns/store_16bit_fp_dp.adoc [hunk 10]
```diff
@@ -10,20 +10,20 @@ Synopsis::
...
 
 {cheri_cap_mode_name} Expansions (RV32)::
-`fsd fs2, offset(cs1')` +
-`fsd fs2, offset(csp)`
+`fsd rs2, offset(cs1')` +
```
Comment (tariqkurd-repo) [inline src/insns/store_16bit_fp_dp.adoc:17]:
```suggestion
`fsd fs2, offset(cs1')` +
```

File: src/insns/store_16bit_fp_dp.adoc [hunk 11]
```diff
@@ -10,20 +10,20 @@ Synopsis::
...
+`fsd rs2, offset(csp)`
```
Comment (tariqkurd-repo) [inline src/insns/store_16bit_fp_dp.adoc:18]:
```suggestion
`fsd fs2, offset(csp)`
```

File: src/insns/store_16bit_fp_dp.adoc [hunk 12]
```diff
@@ -10,20 +10,20 @@ Synopsis::
...
 
 {cheri_int_mode_name} Mnemonics::
-`c.fsd fs2, offset(rs1')` +
-`c.fsd fs2, offset(sp)`
+`c.fsd rs2, offset(rs1')` +
```
Comment (tariqkurd-repo) [inline src/insns/store_16bit_fp_dp.adoc:21]:
```suggestion
`c.fsd fs2, offset(rs1')` +
```

File: src/insns/store_16bit_fp_dp.adoc [hunk 13]
```diff
@@ -10,20 +10,20 @@ Synopsis::
...
+`c.fsd rs2, offset(sp)`
```
Comment (tariqkurd-repo) [inline src/insns/store_16bit_fp_dp.adoc:22]:
```suggestion
`c.fsd fs2, offset(sp)`
```

File: src/insns/store_16bit_fp_dp.adoc [hunk 14]
```diff
@@ -10,20 +10,20 @@ Synopsis::
...
 
 {cheri_int_mode_name} Expansions::
-`fsd fs2, offset(rs1)` +
-`fsd fs2, offset(sp)`
+`fsd rs2, offset(rs1)` +
```
Comment (tariqkurd-repo) [inline src/insns/store_16bit_fp_dp.adoc:25]:
```suggestion
`fsd fs2, offset(rs1)` +
```

File: src/insns/store_16bit_fp_dp.adoc [hunk 15]
```diff
@@ -10,20 +10,20 @@ Synopsis::
...
+`fsd rs2, offset(sp)`
```
Comment (tariqkurd-repo) [inline src/insns/store_16bit_fp_dp.adoc:26]:
```suggestion
`fsd fs2, offset(sp)`
```

File: src/insns/store_32bit_fp.adoc [hunk 16]
```diff
@@ -17,14 +17,14 @@ Synopsis::
 Floating point stores (FSD, FSW, FSH), 32-bit encodings
 
 {cheri_cap_mode_name} Mnemonics::
-`fsd fs2, offset(cs1)` +
-`fsw fs2, offset(cs1)` +
-`fsh fs2, offset(cs1)`
+`fsd rs2, offset(cs1)` +
```
Comment (tariqkurd-repo) [inline src/insns/store_32bit_fp.adoc:20]:
```suggestion
`fsd fs2, offset(cs1)` +
```

File: src/insns/store_32bit_fp.adoc [hunk 17]
```diff
@@ -17,14 +17,14 @@ Synopsis::
...
+`fsw rs2, offset(cs1)` +
```
Comment (tariqkurd-repo) [inline src/insns/store_32bit_fp.adoc:21]:
```suggestion
`fsw fs2, offset(cs1)` +
```

File: src/insns/store_32bit_fp.adoc [hunk 18]
```diff
@@ -17,14 +17,14 @@ Synopsis::
...
+`fsh rs2, offset(cs1)`
```
Comment (tariqkurd-repo) [inline src/insns/store_32bit_fp.adoc:22]:
```suggestion
`fsh fs2, offset(cs1)`
```

File: src/insns/store_32bit_fp.adoc [hunk 19]
```diff
@@ -17,14 +17,14 @@ Synopsis::
...
 
 {cheri_int_mode_name} Mnemonics::
-`fsd fs2, offset(rs1)` +
-`fsw fs2, offset(rs1)` +
-`fsh fs2, offset(rs1)`
+`fsd rs2, offset(rs1)` +
```
Comment (tariqkurd-repo) [inline src/insns/store_32bit_fp.adoc:25]:
```suggestion
`fsd fs2, offset(rs1)` +
```

File: src/insns/store_32bit_fp.adoc [hunk 20]
```diff
@@ -17,14 +17,14 @@ Synopsis::
...
+`fsw rs2, offset(rs1)` +
```
Comment (tariqkurd-repo) [inline src/insns/store_32bit_fp.adoc:26]:
```suggestion
`fsw fs2, offset(rs1)` +
```

File: src/insns/store_32bit_fp.adoc [hunk 21]
```diff
@@ -17,14 +17,14 @@ Synopsis::
...
+`fsh rs2, offset(rs1)`
```
Comment (tariqkurd-repo) [inline src/insns/store_32bit_fp.adoc:27]:
```suggestion
`fsh fs2, offset(rs1)`
```

**Review by arichardson — APPROVED** [review]

**Pull request merged**